atomicwrites/__init__.py

```diff
@@ -19,12 +19,27 @@ def _path_to_unicode(x):
 
 
 if sys.platform != 'win32':
+    def _sync_directory(directory):
+        # Ensure that filenames are written to disk
+        fd = os.open(directory, 0)
+        try:
+            os.fsync(fd)
+        finally:
+            os.close(fd)
+
     def _replace_atomic(src, dst):
         os.rename(src, dst)
+        _sync_directory(os.path.dirname(dst))
 
     def _move_atomic(src, dst):
         os.link(src, dst)
         os.unlink(src)
```

Comment:
Why are you not just using ``os.rename`` here? I don't see a difference besides not being atomic.